content/en/docs/releases.md

```diff
@@ -52,152 +51,166 @@ You can add multiple `release` sections to your project's `rebar.config` under `
+         [<app>],
+         [{dev_mode, true}]}
+       ]}.
+```
+
 Or you can also specify releases with independent configurations:
 
-	 {relx, [{release, {<release name>, "0.0.1"},
-	         [<app>],
-	         [{dev_mode, false},
-	          {include_erts, true}]},
-	        {release, {<release name>, "0.1.0"},
-	         [<app>],
-	         [{dev_mode, true}]}
-	       ]}. 
+```erlang
+{relx, [{release, {<release name>, "0.0.1"},
+         [<app>],
+         [{dev_mode, false},
+          {include_erts, true}]},
+        {release, {<release name>, "0.1.0"},
+         [<app>],
+         [{dev_mode, true}]}
+       ]}.
+```
 You can build specific releases using `rebar3 release -n <release_name>`
 
 ## Developing
 
 While developing you'll likely want all your changes to applications be immediately available in the release. `relx` provides `dev_mode` for this. Instead of copying the applications that make up the release to the release structure it creates symlinks, so compiling and restarting or loading the changed modules, is all that is necessary.
 
-	 {relx, [...
-	        {dev_mode, true},
-	        ...
-	       ]
-	}. 
+```erlang
+{relx, [...
+        {dev_mode, true},
+        ...
+       ]
+}.
 
+```
 
 ## Configuration
 
 ## VM Configuration
 
-
-
 By default `relx` will give a basic `vm.args` file that sets a node name and cookie. For a complete list of options and their use check the [Erlang documentation](http://erlang.org/doc/man/erl.html).
 
-	 ## Name of the node
-	-name {{release_name}}@127.0.0.1
-	
-	## Cookie for distributed erlang
-	-setcookie {{release_name}} 
-To provide a custom `vm.args`, usually placed in the `config/` directory at the root of your project, add this line to the `relx` section of your `rebar.config`:
+```erlang
+## Name of the node
+-name {{release_name}}@127.0.0.1
 
-	 {vm_args, "config/vm.args"} 
+## Cookie for distributed erlang
+-setcookie {{release_name}}
+```
+To provide a custom `vm.args`, usually placed in the `config/` directory at the root of your project, add this line to the `relx` section of your `rebar.config`:
 
+```erlang
+{vm_args, "config/vm.args"} 
+```
 
 ## Application Configuration
 
-
-
 For passing application environment variables there is `sys.config`:
 
-	 [
-	  {<app_name>, [...]}
-	]. 
-The default `sys.config` is empty, so use your own simply add it to your `relx` section of `rebar.config`:
+```erlang
+[
+  {<app_name>, [...]}
+].
+```
 
-	 {sys_config, "config/sys.config"} 
+The default `sys.config` is empty, so use your own simply add it to your `relx` section of `rebar.config`:
 
+```erlang
+{sys_config, "config/sys.config"} 
+```
 
 ## Dynamic Configuration
 
 ##  With OTP-21+ and rebar3 3.6+
 
-
-
 Starting with Erlang/OTP 21 and rebar3 3.6.0 the configuration options `sys_config_src` and `vm_args_src` are available for explicitly including templates that will be rendered at runtime.
 
-	 [
-	  {appname, [{port, ${PORT}}]}
-	]. 
+*sys.config.src*
```

Comment:
Callout : It **seems** there is no way to pass hugo highlight shortcode a filename attr, so this will have to work for now. Could use headers i.e., `### sys.config.src` which makes it stand out a bit more but 🤷. If we want this functionality we'll have to make custom css and a shortcode that takes such an argument.